buildpacks/nodejs-npm-install/CHANGELOG.md

```diff
@@ -10,4 +10,3 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Initial release
- 
```

Comment:
did you mean to remove the new line here?

Reply:
The newline is still there, the removal is of a stray single space character :-)

I opted to include that change in this PR, since it means the changelog contents can be viewed in the PR diff, making it possible to see why a new changelog entry was not added (since the buildpack is new).